### ISSUE TITLE
MY16 cant get it to register

### ISSUE THREAD

**Issue by mikechristiansen** [buxtronix/phev2mqtt#22]
Hi,
This project looks like just what im looking for except I cant get my connection to register.

Ive reset the car leaving both registration slots empty but still don't seem to be able to get a successful register. 

Output from  ./phev2mqtt client register below. 
_mike@ubuntuserver:~/phev2mqtt-master$ ./phev2mqtt client register
INFO[0000] %PHEV_TCP_CONNECTED%                         
Client connected and started!
^C_

And also part of my output from "ip a"
_3: wls16: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 00:c2:c6:a1:d6:0a brd ff:ff:ff:ff:ff:ff
    altname wlp0s16
    inet6 fe80::2c2:c6ff:fea1:d60a/64 scope link 
       valid_lft forever preferred_lft forever_

Is there any way to trouble shoot this? 

Thanks for your time!

**Comment by BunpGhost** [comment]
I'm having a similar issue!
phev2mqtt was working fine until yesterday when I updated to the latest master.
It compiles ok and seems to run ok.

I've tried to register and I get exactly what the OP gets.
So, I tried using the previous MAC ADDRESS that I had spoofed (from my mobile, the old way) and I only get:
TCP_CONNECTED
and nothing more..

what has changed? 
Is it possible to provide the previous version to test? (although I'm not sure what was)

I have a MY14 that was working fine ;)

**Comment by ElektronikCZ** [comment]
I have the same issue with the same symptoms

**Comment by Expaso** [comment]
Yep, same here. MY15 and failing to register in exact the same way,.

**Comment by altserg** [comment]
Please confirm if the issue is fixed with latest patches and close it
